docs/azure_arc_jumpstart/azure_arc_data/aks/aks_postgresql_hyperscale_arm_template_ado/_index.md

```diff
@@ -152,11 +147,9 @@ In this scenario, you will create a new Release pipeline to deploy the environme
 
     ![Screenshot of ARM template parameters config](./14.jpg)
 
-    ![Screenshot of ARM template parameters config](./15.jpg)
-
 * Provide a deployment name.
 
-    ![Screenshot of deployment name config](./16.jpg)
+    ![Screenshot of ARM template parameters config](./15.jpg)
```

Comment:
You have a file jump here. All images should be aligned and in the right order. i.e 14,15,16...

Reply:
Updated